### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
PR raised for Grid, Toast and Rating. Please review

### CODE REVIEW

File: Component/Grid/Grid.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                            <span class="title">Colors</span>
+                        </a>
+                    </li>
+                    <li class="sidebar-li">
+                        <a target="_self" href="/Component/Typography/Typography.html" class="link-active">
+                            <span class="title">Typography</span>
+                        </a>
+                    </li>
+                    <li class="sidebar-li">
+                        <!-- <a target="_self" href="#" class="link-active"> -->
```
Comment (gadgetfather) [inline Component/Grid/Grid.html]: You can remove these if you are not going to use later

File: Component/Image/Image.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.image-container 
+{
+    align-items: center;
+    background-color: var(--component-container-color);
+    /* border: 1px solid black; */
```
Comment (gadgetfather) [inline Component/Image/Image.css:5]: You can remove this also

File: Component/Input/Input.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+{
+  --error-color: rgb(231, 97, 97);
+  --success-color: rgb(74, 168, 74);
+} */
+  .input-container 
+  {
+    align-items: center;
+    background-color: var(--component-container-color);
+    border-radius: 0.3rem;
+    /* border: 1px solid black; */
```
Comment (gadgetfather) [inline Component/Input/Input.css]: This comment also

File: Component/Introduction/getstarted.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    </li>
+  </div>
+  </div>
+  <div class="doc-section">
+    <aside class="aside-style">
+
+      <div class="side-bar">
+        <ul class="top-nav">
+          <li class="sidebar-li">
+            <!-- <a target="_self" href="getstarted.html" class="link-active">   -->
```
Comment (gadgetfather) [inline Component/Introduction/getstarted.html]: Here also , if not needed you can remove it.

File: Component/Modal/Modal.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+                            <span class="title">Colors</span>
+                        </a>
+                    </li>
+                    <li class="sidebar-li">
+                        <a target="_self" href="/Component/Typography/Typography.html" class="link-active">
+                            <span class="title">Typography</span>
+                        </a>
+                    </li>
+                    <li class="sidebar-li">
+                        <!-- <a target="_self" href="#" class="link-active"> -->
```
Comment (gadgetfather) [inline Component/Modal/Modal.html:42]: Here also, comment

File: doc.css [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+    font-size: 2rem;
+    margin-right: 1rem;
+    font-family: cursive;
+    color: var(--secondary-color);
+    text-decoration: none;
+}
+.link-active{
+    text-decoration: none;
+    color:  var(--secondary-color);
+    /* background-color: var(--main-bg-color); */
```
Comment (gadgetfather) [inline doc.css]: You can remove this also

**Review by gadgetfather — APPROVED** [review]
You can remove those comments if not needed. Rest looks good 👍

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go⚡️

**Pull request merged**